examples/hacker_news_assets/hacker_news_assets/repo.py

```diff
@@ -1,8 +1,12 @@
 from dagster import repository, schedule_from_partitions
 
-from .assets import prod_assets, staging_assets
+from .assets import local_assets, prod_assets, staging_assets
 from .jobs.activity_stats import activity_stats_prod_job, activity_stats_staging_job
```

Comment:
The job is not imported!
And the code for the job is still referring to snowflake `SHARED_SNOWFLAKE_CONF` and this is not executable locally.